### ISSUE TITLE
safari - Downloaded prescription as patient but format of file is different

### ISSUE THREAD

**Issue by purushotamk** [prakma/hmp-web#35]
For exmaple, uploaded prescriptionA.txt fie but downloaded on patient side prescriptionA.txt.html file. Now, text file could be opened only in browser

**Comment by prakma** [comment]
I am not sure if this issue really exists. I uploaded a text file as doctor, and saw the same text file downloaded as patient. Did you try this with a specific browser?

**Comment by purushotamk** [comment]
Yes.. it happened when uploaded on chrome and downloaded from safari as patient. Problem does not exist if downloaded from Firefox as patient.So, browser specific

**Comment by purushotamk** [comment]
Cover in FAQ that safari should not be used
